core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/dot-block-editor-settings/dot-block-editor-settings.component.ts

```diff
@@ -36,7 +36,6 @@ export const BLOCK_EDITOR_BLOCKS = [
     { label: 'Heading 3', code: 'heading3' },
     { label: 'Horizontal Line', code: 'horizontalRule' },
     { label: 'Image', code: 'image' },
-    { label: '`Paragraph`', code: 'paragraph' },
```

Comment:
Can you add some test for this?

Reply:
Sure!

Reply:
I'll add a test that checks there is not an item with the `code: 'paragraph'` in the options.